test/e2e/tests/bridge/constants.ts

```diff
@@ -1,6 +1,6 @@
 import type { FeatureFlagResponse } from '@metamask/bridge-controller';
 
-export const EXPECTED_MAINNET_ETH_BALANCE = '23.9999';
```

Comment:
balance after tx's is changed slightly as the gas we used before was legacy, now is eip1559